spanner/spanner_snippets/snippet.go

```diff
@@ -1097,25 +1098,57 @@ func updateUsingDMLStruct(ctx context.Context, w io.Writer, client *spanner.Clie
 // [START spanner_dml_getting_started_insert]
 
 func writeUsingDML(ctx context.Context, w io.Writer, client *spanner.Client) error {
-	_, err := client.ReadWriteTransaction(ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
-		stmt := spanner.Statement{
-			SQL: `INSERT Singers (SingerId, FirstName, LastName) VALUES
+	_, err := client.ReadWriteTransaction(
+		ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
```

Comment:
I think I preferred it being on the same line. Generally try to avoid breaking in the middle of function calls (not including struct literals).